.gitattributes

```
@@ -1 +1,3 @@
 * text=auto
+test/e2e/fixtures/explore/blocks/README.md text
+test/e2e/fixtures/explore/blocks/Qm* binary
```

- we expect CARs and CIDv1 blocks in the future, let's add them here
- `linguist-generated` will collapse fixture data, making PRs with them  easier to review

```suggestion
test/e2e/fixtures/**/Qm* binary linguist-generated
test/e2e/fixtures/**/b* binary linguist-generated
test/e2e/fixtures/**/*.raw binary linguist-generated
test/e2e/fixtures/**/*.car binary linguist-generated
```